Microsoft/WebTemplateStudio
# App Service - update generated README

Updated README in generated project to include more details.

To test:
- Create project (please test different combination of frontend and backend frameworks and pages)
- Make sure that project is successfully created
- Check that the README is generated in the project
